test-data/unit/lib-stub/dataclasses.pyi

```diff
@@ -0,0 +1,36 @@
+from typing import Any, Callable, Optional, TypeVar, overload
+
+_T = TypeVar('_T')
+_C = TypeVar('_C', bound=type)
+
+
+@overload
+def dataclass(_cls: _C,
+              *,
+              init: bool = ...,
+              repr: bool = ...,
+              eq: bool = ...,
+              order: bool = ...,
+              unsafe_hash: bool = ...,
+              frozen: bool = ...) -> _C: ...
+
+
+@overload
+def dataclass(_cls: None = ...,
+              *,
+              init: bool = ...,
+              repr: bool = ...,
+              eq: bool = ...,
+              order: bool = ...,
+              unsafe_hash: bool = ...,
+              frozen: bool = ...) -> Callable[[_C], _C]: ...
+
+
+def field(*,
```

Comment:
It is good that you added this to `lib-stub`. Be sure to make updates here, if https://github.com/python/typeshed/pull/1944 gets updated.